### ISSUE TITLE
[FEATURE] This is fire! Keep it up!

### ISSUE THREAD

**Issue by ItsCEED** [triat/yogi-bot#5]
This is amazing! if you need front end design im willing to do it!

Also, would be this PUG/MIX/AUTOMIX doable in the future?

**Comment by triat** [comment]
Hey @ItsCEED,
Thanks for proposition your help, it is always appreciated.

Right now, the project is still at a very early stage, things are still moving really fast and talking about design is maybe a bit too soon.

But as I'm a terrible frontend developer, I will love to get some help on that. Feel free to contribute/comment! 

Regarding your question about PUG/MIX/AUTOMIX ... could be, I haven't thought about it. It needs to be defined what would be the features of it but I guess everything is doable (as a module that you can install/activate maybe).

**Comment by ItsCEED** [comment]
Hey @triat, im not a front end developer but a designer :(

So if someone is willing to code the front end i can work on a template and further modification once the base code for it is done.

About the PUG/MIX/AUTOMIX/FACEIT LIKE platform module, i can give you direction in the future if you are interested to develope. To be honest you can even profit from it or from further extensions on it, like keep the base free, but sell some modification like key parts like auto ban on leave or something around those lines.

**Comment by triat** [comment]
Oh .. sorry my mistake, I miss understood what you were telling me :) 
I guess I can code it as I've done it several time before but this is not where I'm the best :) 

Regarding your proposition for the platform, I'll keep it as a nice to have for the future, as it is not the plan for now and I have already quite a lot of job to do before I reach an equivalent to eBot.
